python/cudf/cudf/_lib/lists.pyx

```diff
@@ -126,7 +126,7 @@ def sort_lists(Column col, bool ascending, str na_position):
     return Column.from_unique_ptr(move(c_result))
 
 
-def extract_element(Column col, size_type index):
+def extract_element_scalar(Column col, int index):
```

Comment:
Why did this change? Not a problem but it seems extraneous.

Reply:
Fixed